public/css/normalize.css

```diff
@@ -0,0 +1,444 @@
+/*! normalize.css v3.0.2 | MIT License | git.io/normalize */
```

Comment:
Are normalize and skeleton copied from somewhere / generated from something? If we want to add additional styling, will we do it in one of these or do it in separate files? In general how easy is it to build upon?

Reply:
Yes they are downloaded from [Skeleton's](http://getskeleton.com) website. I would imagine we would not try to touch those stylesheets, and make our own if it's a septette styling. In general I think it depends on the scope, but I would say using `themes` and CSS in JS are the new thing now, but it's probably an overkill here. 

In general I imagine the app doesn't require too much customization and changes, so modifying the Skeleton CSS could be an option too, though not if we're going to heavily change it

Reply:
Sounds good - since it's from an external source let's keep it as is and when we need to add new styling we'll make a separate stylesheet.